### DISCUSSION TITLE
How to print schema?

### DISCUSSION THREAD

**Discussion by ghost** [SeaQL/sea-orm#940] [Q&A] [answered]
After reading this
```
let schema_manager = SchemaManager::new(db); // To investigate the schema
```
I was wondering if there is a way to print the whole schema?

**Comment by billy1624** [comment]
Hey @JediMaster25, welcome! Do you mean schema discovery like https://github.com/SeaQL/sea-schema#schema-discovery?

**Comment by ghost** [comment]
That seems difficult to read. I was thinking of something like `create_bakery_table().to_string(PostgresQueryBuilder)` to print the whole database schema.

Or printing the [graphql schema](https://github.com/stashapp/stash-box/tree/master/graphql/schema) like stash-box does based on it's [database](https://github.com/stashapp/stash-box/tree/master/pkg/database/migrations/postgres).

edit: This is pretty much what I was looking for: [Generate Entity from Database](https://www.sea-ql.org/sea-orm-tutorial/ch01-04-entity-generation.html#generate-entity-from-database)

**Comment by billy1624** [comment, marked as answer]
Alright! Please check our documentation for generating SeaORM entity file at https://www.sea-ql.org/SeaORM/docs/generate-entity/sea-orm-cli/#generating-entity-files
